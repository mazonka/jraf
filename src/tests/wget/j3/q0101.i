command=jw md 0 /aaa : read 0 / : read 0 /.jraf.ver/