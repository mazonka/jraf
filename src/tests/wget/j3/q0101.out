OK /aaa : OK 1 -1 1 1 -1 aaa : JRAF_FAIL system path